transbank/webpay/webpayplus/transaction.py

```diff
@@ -35,19 +35,17 @@ def create(cls, buy_order, session_id, amount,
             "return_url": return_url,
         })
 
-        http_client = WebpayPlus.http_client
         final_url = base_url + cls.CREATE_TRANSACTION_ENDPOINT
 
         try:
-            http_response = http_client.post(final_url, data=payload, headers=headers)
+            http_response = requests.post(final_url, data=payload, headers=headers)
```

Comment:
This change doesn't seem to be related to the commit message ("Reorder options individually for each transaction").